templates/CRM/Tag/Page/Tag.tpl

```diff
@@ -410,14 +410,14 @@
   li.is-reserved > a:after {
     content: ' *';
   }
-  {/literal}{if !call_user_func(array('CRM_Core_Permission', 'check'), 'administer reserved tags')}{literal}
+  {/literal}{crmPermission has='administer reserved tags'}{literal}
```

Comment:
original was if _doesn't_ have permission

Reply:
oops, fixed it now :)